leases/tests/test_lease_mutations.py

```diff
@@ -469,14 +454,448 @@ def test_update_berth_lease_application_already_has_lease(
+              id
+            }
+            status
+            comment
+            place {
+                id
+            }
+            application {
+                id
+                status
+            }
+        }
+    }
+}
+"""
+
+
+@pytest.mark.parametrize(
+    "api_client", ["berth_services", "berth_handler"], indirect=True,
+)
+def test_create_winter_storage_lease(
+    api_client, winter_storage_application, winter_storage_place, customer_profile
+):
+    winter_storage_application.customer = customer_profile
+    winter_storage_application.save()
+
+    variables = {
+        "applicationId": to_global_id(
+            WinterStorageApplicationNode, winter_storage_application.id
+        ),
+        "placeId": to_global_id(WinterStoragePlaceNode, winter_storage_place.id),
+    }
+
+    assert WinterStorageLease.objects.count() == 0
+
+    executed = api_client.execute(CREATE_WINTER_STORAGE_LEASE_MUTATION, input=variables)
+
+    assert WinterStorageLease.objects.count() == 1
+
+    assert (
+        executed["data"]["createWinterStorageLease"]["winterStorageLease"].pop("id")
+        is not None
+    )
+    assert executed["data"]["createWinterStorageLease"]["winterStorageLease"] == {
+        "status": "DRAFTED",
+        "startDate": str(calculate_winter_storage_lease_start_date()),
+        "endDate": str(calculate_winter_storage_lease_end_date()),
+        "comment": "",
+        "boat": None,
+        "customer": {
+            "id": to_global_id(ProfileNode, winter_storage_application.customer.id)
+        },
+        "application": {
+            "id": variables.get("applicationId"),
+            "status": ApplicationStatus.OFFER_GENERATED.name,
+        },
+        "place": {"id": variables.get("placeId")},
+    }
+
+
+@pytest.mark.parametrize(
+    "api_client",
+    ["api_client", "user", "harbor_services", "berth_supervisor"],
+    indirect=True,
+)
+def test_create_winter_storage_lease_not_enough_permissions(
+    api_client, winter_storage_application, winter_storage_place
+):
+    variables = {
+        "applicationId": to_global_id(
+            WinterStorageApplicationNode, winter_storage_application.id
+        ),
+        "placeId": to_global_id(WinterStoragePlaceNode, winter_storage_place.id),
+    }
+
+    executed = api_client.execute(CREATE_WINTER_STORAGE_LEASE_MUTATION, input=variables)
+
+    assert_not_enough_permissions(executed)
+
+
+def test_create_winter_storage_lease_application_doesnt_exist(
+    superuser_api_client, winter_storage_place
+):
+    variables = {
+        "applicationId": to_global_id(WinterStorageApplicationNode, randint(0, 999)),
+        "placeId": to_global_id(BerthNode, winter_storage_place.id),
+    }
+
+    executed = superuser_api_client.execute(
+        CREATE_WINTER_STORAGE_LEASE_MUTATION, input=variables,
+    )
+
+    assert_doesnt_exist("WinterStorageApplication", executed)
+
+
+def test_create_winter_storage_place_lease_winter_storage_place_doesnt_exist(
```

Comment:
first mention of `place` is a typo?

Reply:
It's what happens when you `find/replace` without double checking 😅